jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
```

Comment:
> Wouldn't you potentially get repeat commit messages if you were listening to both multi-dev environments, and then dev once those commits get merged into dev?

I see that as a feature, not a bug. The message can/should indicate what env it's coming from, making it easy to follow the progress in JIRA. If people explicitly don't want this, they can limit to dev, or use `first-parent` like the dashboard. 

Definitely no to branch-checking.